Sources/FDB/Future/Future+Bytes.swift

```diff
@@ -48,7 +48,7 @@ extension FDB.Future {
     /// Parses key bytes result from current future
     ///
     /// Warning: this should be only called if future is in resolved state
-    func parseKeyBytes() throws -> Bytes {
+    internal func parseKeyBytes() throws -> Bytes {
```

Comment:
Bringing this to your attention — I'm assuming this is not allowed re: SemVer, but I accidentally introduced this as public in #59, failing to notice the "value" version was not.

Reply:
It doesn't matter, `FDB.Future` class is internal already, and also in modules `public` should be stated explicitly, otherwise it's `internal` (might be mistaken, but from my experience this is how it is).

Reply:
I'll leave it with `internal` for consistency then.